test/test-apppath.js

```diff
@@ -7,34 +7,34 @@ var assert  = yeoman.assert;
 var fs      = require('fs');
 var test    = require('./helper.js');
 
+var config = [
+  '{',
+  '  "generator-backbone": {',
+  '    "appPath": "public",',
+  '    "appName": "Temp"',
+  '  }',
+  '}'
+].join('\n');
```

Comment:
Instead of constructing JSON this way wouldn't it be better just to JSON.stringify() an object?